### ISSUE TITLE
 expose port 8529 to host port 80 when using arangodb starter

### ISSUE THREAD

**Issue by weiklr** [arangodb/arangodb#13052]
Hi, is it possible to expose the coordinator port 8529 via the host port 80 when using arangodb start to create clusters?

I tried using the -p and --expose flags with the docker run command but it doesn't work.

## My Environment

* __ArangoDB Version__:      3.7.3
* __Storage Engine__:             RocksDB
* __Deployment Mode__:       Cluster
* __Deployment Strategy__:   ArangoDB Starter in Docker
* __Configuration__:               
* __Infrastructure__:               own
* __Operating System__:        RHEL 7.4

**Comment by dothebart** [comment]
Hi, 
you can use the [pass through options](https://www.arangodb.com/docs/stable/programs-starter-options.html#passing-through-other-database-options) to specify the [server.endpoint](https://www.arangodb.com/docs/stable/programs-arangod-options.html).

**Comment by weiklr** [comment]
Thanks! I will give it a shot!

**Comment by weiklr** [comment]
Don't think it works. I got this error: Option '--server.endpoint' is essential to the starters behavior and cannot be overwritten.

**Comment by dothebart** [comment]
Sorry, yes you are correct. if you specify `--starter.port`, the server will be on n+1

**Comment by weiklr** [comment]
I don't think that will work for me, as I only want to bind port 80 to 8529 and have the remaining ports remain as they are. Is it possible to expose 8529 in the arango starter dockerfile so that the host port can be mapped?

**Comment by dothebart** [comment]
yes, you always can use `-p80:8529` to map that port into the docker container.

**Comment by weiklr** [comment]
Yup I tried that before I created this question. I can't access the arango web ui via port 80 on my browser. I have to navigate to localhost:8529 still to access the web ui. I attaching my docker ps logs if it helps. The port gets binded on the arangodb-starter image, but it doesn't get binded to the actual arangodb image. 

[docker-ps-arangodb-log.txt](https://github.com/arangodb/arangodb/files/5577879/docker-ps-arangodb-log.txt)
